tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/ResponseFactory.cs

```diff
@@ -0,0 +1,38 @@
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.Responses.Package;
+
+namespace Azure.Sdk.Tools.Cli.Helpers;
+
+/// <summary>
+/// Factory for creating standardized command responses.
+/// </summary>
+public class ResponseFactory : IResponseFactory
+{
+    /// <inheritdoc />
+    public PackageOperationResponse CreateFailureResponse(string message, PackageInfo? packageInfo = null, string[]? nextSteps = null)
+    {
+        return new PackageOperationResponse
```

Comment:
Lets make these helper methods on PackageOperationResponse instead of creating this ResponseFactor and interface.

Reply:
Refactored them as extension methods.